include/seqan3/alphabet/composite/alphabet_variant.hpp

```diff
@@ -126,8 +126,8 @@ inline bool constexpr one_alternative_is<union_composition<alternatives...>,
 template <typename ... alternatives,
           template <typename> typename fun_t,
           typename target_t>
-    requires tuple_size_concept<target_t> && !cartesian_composition_concept<target_t>
-inline bool constexpr one_alternative_is<union_composition<alternatives...>,
+    requires tuple_size_concept<target_t> && !alphabet_tuple_base_concept<target_t>
```

Comment:
can you directly rename the concept to camel case?

Reply:
I think we need to discuss whether helper concepts that are in detail should also follow this convention. I am kind of against it... exposition-only types in the standard are also not CamelCase.